src/components/Tabs/Tab.tsx

```diff
@@ -171,7 +171,7 @@ const Tab = <T extends ElementType = 'button'>(
         {...props}
       >
         {children}
-        {counter ? (
+        {typeof counter === 'number' ? (
```

Comment:
Yeah but typing isn't that strong so far, probably because of Tab forward, you can pass null/false/whatever to the counter so I used that to be safer, otherwise we may find a way to fix the typing

Reply:
Arf alright then lets go like this